core/silkworm/state/state.hpp

```diff
@@ -17,16 +17,14 @@
 #ifndef SILKWORM_STATE_STATE_HPP_
 #define SILKWORM_STATE_STATE_HPP_
 
-#include <optional>
-
-#include <silkworm/common/base.hpp>
 #include <silkworm/types/account.hpp>
-#include <silkworm/types/block.hpp>
 #include <silkworm/types/receipt.hpp>
 
+#include "block_state.hpp"
```

Comment:
Perhaps
```
#include <silkworm/state/block_state.hpp>
```
instead for extra clarity and consistency?

I personally prefer to use relative "..." includes only in .cpp files, but full paths in headers.

Reply:
ok!